test/src/sdk-harness/build.sh

```diff
@@ -27,7 +27,7 @@ while true; do
   fi
 done
 
-test_dirs="${base_dir}/test_artifacts/* ${base_dir}/test_projects/*"
+test_dirs="${base_dir}/test_artifacts/* ${base_dir}/test_projects/* ${base_dir}/test_artifacts/storage_vec/*"
```

Comment:
well im gonna have like 10 more projects there for every type, im just finalizing what tests there will be for now so removed the other types. If i moved all the projects for all the types up one, it would be kinda crowded

Reply:
Pros and cons: having to add sub-directories to this line is extra work (and also a manual step). Putting everything flat could make it crowded. I'm actually okay with a crowded flat directory personally, and e.g. the sway e2e tests are flat (other than pass/fail).

Reply:
any way for me to make that line more generic so its not specific to storage_vec? Like for future folders the same thing being usable

Reply:
Personally I just think that all contracts should be put into their own seperate folders inside the test_projects folder, so that it'll be more organised. With more and more tests and more and more complicated features theres gonna be way too many folders if we dont organise.

Plus that way maybe some generic bash command could be written too

Reply:
I think you can maybe use `**` (ref: https://stackoverflow.com/a/28199633) for this. Feel free to file an issue and save this for future work.